h2d/Font.hx

```diff
@@ -279,7 +287,7 @@ class Font {
 		@param size The new font size.
 	**/
 	public function resizeTo( size : Int ) {
-		var ratio = size / initSize;
+		var ratio = size / this.size;

```

Comment:
Why is this change needed? It makes initSize unused, so if this really is correct that should be removed.